audit-triplestore/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,7 +12,11 @@
      <cm:default-properties>
        <cm:property name="jms.brokerUrl" value="tcp://localhost:61616"/>
        <cm:property name="jms.fcrepoEndpoint" value="topic:fedora"/>
-       <cm:property name="triplestore.baseUrl" value="localhost:3030/ds/test"/>
+       <cm:property name="triplestore.baseUrl" value="localhost:3030/fedora4"/>
```

Comment:
Is there any benefit to having two properties?  Why not just make triplestore.baseUrl localhost:3030/fedora4/update?